### PR TITLE
Updated proof-request logic to the new, non-deprecated APIs

### PR DESCRIPTION
**Pull request by esune**
This PR updates ICOB self-verify functionality to use the new, non-deprecated APIs on the agent admin interface. For the change to be fully-functional, a new version of the agent that includes the changes in [this PR](https://github.com/hyperledger/aries-cloudagent-python/pull/222) needs to be used. 

The PR should be moved out of draft mode once the new version of aca-py is available and the change can be pulled in ICOB.

### CODE REVIEW

File: starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py [hunk 1]
```diff
@@ -279,9 +277,7 @@ def verify(self, request, pk=None):
                 result = {
                     "success": True,
                     "result": {
-                        "presentation_request": presentation_state[
-                            "presentation_request"
-                        ],
+                        "presentiation_request": presentation_state["presentation_request"],
```
Comment (andrewwhitehead) [inline starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py]: Minor typo

**Comment by esune** [comment]
Waiting for #354 to be merged before updating this, as some changes will be required per @andrewwhitehead

**Comment by WadeBarnes** [comment]
@esune, #354 has been merged.

**Comment by esune** [comment]
@WadeBarnes thanks, I am working on updating the logic to support the code added by the PR

**Comment by esune** [comment]
@andrewwhitehead when you have time, can you please check everything looks good and merge?

File: credential-registry/client/tob-web/src/app/data-types.ts [hunk 2]
```diff
@@ -215,50 +211,50 @@ export namespace Model {
...
       let ret = [];
-      if(typeof this.result === 'object' && this.result.presentation) {
+      if (typeof this.result === "object" && this.result.presentation) {
         let attrs = this.result.presentation.requested_proof.revealed_attrs;
-        for(let k in attrs) {
-          ret.push({name: k, value: attrs[k].raw});
+        for (let k in attrs) {
+          const attributeName = this.result.presentation_request.requested_attributes[k].name;
+          ret.push({ name: attributeName, value: attrs[k].raw });
         }
```
Comment (esune) [inline credential-registry/client/tob-web/src/app/data-types.ts]: This is the only change in this file, the rest is just the code being formatted.
Do we have standard formatting settings for our JS/TS code?

**Pull request merged**